src/components/page-configuration/change-repository-form.js

```diff
@@ -64,162 +65,174 @@ export const ChangeRepositoryForm = (props) => {
   };
 
   return (
-    <Accordion.Item className="accordion__item">
-      <Accordion.Header className="accordion__header body_short">
-        Change GitHub repository
+    <Accordion.Item className="accordion">
+      <Accordion.Header>
+        <Typography>Change GitHub repository</Typography>
       </Accordion.Header>
-      <Accordion.Panel className="accordion__panel">
-        <form onSubmit={handleSubmit} className="accordion__content">
-          {saveState.status === requestStates.FAILURE && (
-            <Alert type="danger" className="gap-bottom">
-              Failed to change repository. {saveState.error}
-            </Alert>
-          )}
-          <p className="body_short">
-            Full URL, e.g. 'https://github.com/equinor/my-app'
-          </p>
-          <Input
-            disabled={
-              updateRepositoryProgress ||
-              saveState.status === requestStates.IN_PROGRESS
-            }
-            type="url"
-            value={repository}
-            onChange={(ev) => setRepositoryAndResetSaveState(ev.target.value)}
-          />
-          <div className="o-action-bar">
+      <Accordion.Panel>
+        <div className="grid grid--gap-medium">
+          <form onSubmit={handleSubmit} className="grid grid--gap-medium">
+            {saveState.status !== requestStates.FAILURE && (
+              <div>
+                <Alert type="danger">
+                  Failed to change repository. {saveState.error}
+                </Alert>
+              </div>
+            )}
+            <TextField
+              disabled={
+                updateRepositoryProgress ||
+                saveState.status === requestStates.IN_PROGRESS
+              }
+              type="url"
+              value={repository}
+              onChange={(ev) => setRepositoryAndResetSaveState(ev.target.value)}
+              label="URL"
+              helperText="e.g. 'https://github.com/equinor/my-app'"
+            />
             {(updateRepositoryProgress ||
               saveState.status === requestStates.IN_PROGRESS) && (
-              <>
-                <CircularProgress size="24" />
-                <span className="progress">Updating…</span>
-              </>
+              <div>
+                <CircularProgress size="20" /> <span>Updating…</span>
+              </div>
             )}
             {!updateRepositoryProgress &&
               saveState.status !== requestStates.IN_PROGRESS && (
```

Comment:
suggestion: use `short-if` and remove unnecessary `<span>`
```
            {(updateRepositoryProgress ||
              saveState.status === requestStates.IN_PROGRESS) ? (
              <div>
                <CircularProgress size="20" /> <span>Updating…</span>
              </div>
            ) : (
```

Reply:
since there are multiple checks here I went with just removing the unnecessary <span>